code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Dispenser.dm

```diff
@@ -179,41 +181,21 @@
 	if (adj_temp < 0 && M.bodytemperature > targ_temp)
 		M.bodytemperature = min(targ_temp, M.bodytemperature - (adj_temp * TEMPERATURE_DAMAGE_COEFFICIENT))
 
-/singleton/reagent/alcohol
```

Comment:
I noticed this was deleted. Ethanol is used in multiple medicines via the Chemical Dispenser in the Pharmacy, but this part causes it to not show up. When spawned in by admins, it has no name and defines itself as "null".

Reply:
Yea someone seems to have reused this class (that says to not spawn, aka virtual) for ethanol instead of making a subclass of it